src/openforms/prefill/api/views.py

```diff
@@ -67,3 +76,73 @@ def get_objects(self):
         choices = plugin.get_available_attributes()
 
         return [ChoiceWrapper(choice) for choice in choices]
+
+
+@extend_schema(
+    summary=_("List available objecttypes for Objects API"),
+    parameters=[OBJECTS_API_GROUP_QUERY_PARAMETER],
+)
+class PluginObjectsAPIObjecttypeListView(ObjecttypesListView):
+    """
+    List the available prefill objecttypes for Objects API plugin.
+    """
+
+    pass
+
+
+@extend_schema(
+    summary=_("List available objecttype versions for Objects API"),
+    parameters=[OBJECTS_API_GROUP_QUERY_PARAMETER],
+)
+class PluginObjectsAPIObjecttypeVersionListView(ObjecttypeVersionsListView):
+    """
+    List the available prefill objecttype versions for Objects API plugin.
+    """
+
+    def get_objects(self):
```

Comment:
I don't get why the implementation is overridden here?

Reply:
It was weird and I forgot to ask this in this PR. I saw that the same view in the registrations uses `submission_uuid` which I don't get and that's the only reason I did it.

https://github.com/open-formulieren/open-forms/blob/master/src/openforms/registrations/contrib/objects_api/api/views.py#L91

In the client:
https://github.com/open-formulieren/open-forms/blob/master/src/openforms/contrib/objects_api/clients/objecttypes.py#L56

Reply:
Discussed this and it is a copy paste mistake, so the name will be updated accordingly.